numpy/random/_generator.pyx

```diff
@@ -1732,33 +1735,46 @@ cdef class Generator:
         ...                    7515, 8230, 8770])
 
         Does their energy intake deviate systematically from the recommended
-        value of 7725 kJ?
-
-        We have 10 degrees of freedom, so is the sample mean within 95% of the
-        recommended value?
+        value of 7725 kJ? Our null hypothesis will be the absence of deviation,
+        and the alternate hypothesis will be the presence of an effect that could be
+        either positive or negative, hence making our test 2-tailed. 
+
+        Because we are estimating the mean and we have N=11 values in our sample,
+        we have N-1=10 degrees of freedom [3]_. We set our signifance level to 95% and 
+        compute the t statistic using the empirical mean and standard deviation
+        of our intake, setting the ddof parameter to the unbiased
+        value so the divisor in the standard deviation will be degrees of
+        freedom.
 
-        >>> s = np.random.default_rng().standard_t(10, size=100000)
         >>> np.mean(intake)
         6753.636363636364
         >>> intake.std(ddof=1)
         1142.1232221373727
+        >>> t = (np.mean(intake)-7725)/(intake.std(ddof=1)/np.sqrt(len(intake)))
+        >>> t
+        -2.8207540608310198
 
-        Calculate the t statistic, setting the ddof parameter to the unbiased
-        value so the divisor in the standard deviation will be degrees of
-        freedom, N-1.
+        We draw 1000000 samples from Student's t distribution with the adequate
+        degrees of freedom.
 
-        >>> t = (np.mean(intake)-7725)/(intake.std(ddof=1)/np.sqrt(len(intake)))
         >>> import matplotlib.pyplot as plt
+        >>> s = np.random.default_rng().standard_t(10, size=1000000)
         >>> h = plt.hist(s, bins=100, density=True)
 
-        For a one-sided t-test, how far out in the distribution does the t
-        statistic appear?
+        Does our t statistic land in one of the two critical regions found at 
+        both tails of the distribution?
 
         >>> np.sum(s<t) / float(len(s))
```

Comment:
If a 2-sided test, it is better to use np.sum(np.abs(s) < np.abs(t))

Reply:
Indeed! I Will update accordingly.

Reply:
actually, if using abs, those values should be swapped, np.sum(np.abs(t) < np.abs(s))